target/classes/static/index.html

```diff
@@ -0,0 +1,75 @@
+<html>
+  <head>
+    
+    <title>Hyperlocal Spots</title>
+    
+    <script async src="https://microapps.google.com/apis/v1alpha/microapps.js"></script>
+
+	<!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+
+  </head>
+
+  <body>
+
+    <nav class="navbar navbar-expand-sm bg-dark justify-content-center navbar-dark">
+      <a class="navbar-brand" href="#">HyperLocal Market</a>
+    </nav>
+
+    <br>    
+		
+    <div style="width:300px; margin:0 auto;">    
+       <ul class="list-group">
+	    <li class="list-group-item">		
+	    	    <div class="card bg-info text-center text-white" style="width:250px">
+			    <div class="Card-body">
+            Litchi
+			    </div>
+		    </div>
+	    </li>
+	    <li class="list-group-item">
+		    <div class="card text-center bg-info text-white" style="width:250px">
+			    <div class="Card-body">
+            Mango
+			    </div>
+		    </div>
+	    </li>
+	    <li class="list-group-item">
+		    <okay>
+		    <div class="card text-center bg-info text-white" style="width:250px">
+			    <div class="Card-body">
+				    Apple
+			    </div>
+  		    </div>
+  	    </li>
+      </ul>
+    </div>
+    <br>
+    <br>
+    <div>
+      <form action="/publish" method="post">
+        Publish message: <input type="text" name="message" />
+        <input type="submit" value="Push to PubSub"/>
+      </form>
+    </div>
+  </body>
+
+  <script>
+    const request = {nonce: 'ahskid'};
+    microapps.getIdentity(request).then(response => {
+      const decodedIdentity = JSON.parse(atob(response.split('.')[1]));
+      console.log("getIdentityResponse", decodedIdentity);
+    }).catch(error => {
+      console.error("Error happened: ", error);
+    });
```

Comment:
Is this code working or required ?

Reply:
Ok. Removed that too. Changed it to a simple index.html file